docs/markdown/Reference-manual.md

```diff
@@ -402,6 +402,10 @@ the following special string substitutions:
 - `@PLAINNAME@`: the input filename, without a path
 - `@BASENAME@`: the input filename, with extension removed
 - `@PRIVATE_DIR@` *(since 0.50.1)*: path to a directory where the custom target must store all its intermediate files.
+- `@SOURCE_ROOT@`: the full path to the root of the source tree.
+- `@BUILD_ROOT@`: the full path to the root of the build tree.
+- `@CURRENT_SOURCE_DIR@`: this is the directory where the currently
+  processed meson.build is located in.
```

Comment:
This should probably mention that it is a relative path from the cwd where the target was executed.

Reply:
That's another weirdness we should eventually tackle some day. With vs backend they are absolute paths, I don't know why. But documenting it as `the full path` is indeed misleading.

Reply:
Maybe something like "depending on the backend, this may be an absolute or a relative path" then?

Reply:
done.